dapr/proto/runtime/v1/dapr.proto

```diff
@@ -255,6 +268,49 @@ message UnregisterActorTimerRequest {
   string name = 3;
 }
 
+// RegisterActorReminderRequest is the message to register a reminder for an actor of a given type and id.
+message RegisterActorReminderRequest {
+  string actor_type = 1;

```

Comment:
I wonder if we can use `Actor` protobuf definition. it is in internals gRPC proto but we can move this to common and then use `actor_type` and `actor_id` with `Actor` protobuf message. 

https://github.com/dapr/dapr/blob/4fa27f979924c1dfb20995196e3285710b05bddd/dapr/proto/internals/v1/service_invocation.proto#L34-L41

What do you think?

Reply:
IMO, this is more complicated API. It will require users to instantiate a new builder for the Actor type too. I think the flat structure is easier to use.